uxarray/core/grid.py

```diff
@@ -319,6 +320,31 @@ def compute_face_areas(self, quadrature_rule="triangular", order=4):
 
         return self._face_areas
 
+    def equals(self, other):
```

Comment:
Implementing this functionality using ``__eq__(self, other):`` and ``__ne__(self, other):`` methods would be more practical 

```Python
# instead of 
uxds_01.uxgrid.equals(uxds_02.uxgrid)
# can check
uxds_01.uxgrid == uxds_02.uxgrid
# or alternatively 
uxds_01.uxgrid != uxds_02.uxgrid